data_validation/query_builder/query_builder.py

```diff
@@ -220,6 +220,8 @@ def compile(self, ibis_table):
         comparison_field = ibis_table[self.field_name]
         alias = self.alias or self.field_name
         if self.cast:
+            # TODO Not sure what to do here yet.
+            # if self.cast != consts.CONFIG_CAST_UUID_STRING:
```

Comment:
Is this comment still needed, such as is it still gonna be addressed on this PR?

Reply:
Great spot, that should have been removed. Thanks